packages/client/src/client/notifications.web.ts

```diff
@@ -99,11 +114,12 @@ export async function registerFxNotifications() {
 }
 
 let quotesReceivedSubscription: Subscription | null = null
+let acceptedRfqWithQuoteSubscription: Subscription | null = null
+let createdCreditSubscription: Subscription | null = null
 
-export async function registerCreditQuoteNotifications() {
+export async function registerCreditNotifications() {
```

Comment:
would have to be here where we check `fin.Application.getApplications` or whatever it is and do the logic to see if Launcher or Workspace is running when RT Credit starts up ... prob need to do for FX as well, with moving to `LiveRatesCore`